Formula/otf2.rb

```diff
@@ -23,28 +23,14 @@ class Otf2 < Formula
   depends_on "sphinx-doc" => :build
   depends_on "gcc" # for gfortran
   depends_on "open-mpi"
-  depends_on "python@3.9"
-
-  resource "future" do
-    url "https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz"
-    sha256 "b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d"
-  end
+  depends_on "python@3.10"
+  depends_on "six"
 
   def install
-    python3 = Formula["python@3.9"].opt_bin/"python3"
-    xy = Language::Python.major_minor_version python3
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{xy}/site-packages"
-
-    resource("future").stage do
-      system python3, *Language::Python.setup_install_args(libexec/"vendor")
```

Comment:
Not sure if we need `future`. It is listed in `INSTALL` as part of Python 2 + 3 compatibility layer. However, the main usage is `builtins`, which is a no-op on Python 3. I didn't see any `import future` or `from future ...` that would require `future` package.

On other hand, we do need `six`. Looks like existing bottles lack Python bindings because we didn't include `six`.

Reply:
Formula/otf2.rb